app/src/main/java/ru/flicksbox/movie/presentation/FavouritesViewHolder.kt

```diff
@@ -0,0 +1,34 @@
+package ru.flicksbox.movie.presentation
+
+import android.net.Uri
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.squareup.picasso.Picasso
+import ru.flicksbox.R
+import ru.flicksbox.movie.domain.MovieEntity
+import ru.flicksbox.movie.domain.TVShowEntity
+import ru.flicksbox.utils.buildImageUrl
+
+class FavouritesViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    private val imageView = itemView.findViewById<ImageView>(R.id.film_image)
+
+    fun bind(movie: MovieEntity?, tvShow: TVShowEntity?) {
```

Comment:
Remove dependency on **Entities** here
UI **must be completely independent** from business models and logic